Mac/BuildScript/scripts/postflight.patch-profile

```diff
@@ -77,16 +77,17 @@ bash)
 	fi
 	;;
 fish)
-	CONFIG_DIR="${HOME}/.config/fish"
-	RC="${CONFIG_DIR}/config.fish"
+	CONFIG_DIR="${HOME}/.config/fish/conf.d/"
+	RC="${CONFIG_DIR}/python-${PYVER}.fish"
 	mkdir -p "$CONFIG_DIR"
 	if [ -f "${RC}" ]; then
 		cp -fp "${RC}" "${RC}.pysave"
 	fi
-	echo "" >> "${RC}"
-	echo "# Setting PATH for Python ${PYVER}" >> "${RC}"
-	echo "# The original version is saved in ${RC}.pysave" >> "${RC}"
-	echo "set -x PATH \"${PYTHON_ROOT}/bin\" \"\$PATH\"" >> "${RC}"
+	echo "# Setting PATH for Python ${PYVER}" > "${RC}"
+	if [ -f "${RC}.pysave" ]; then
+		echo "# The original version is saved in ${RC}.pysave" >> "${RC}"
+	fi
+	echo "fish_add_path -g \"${PYTHON_ROOT}/bin\"" >> "${RC}"
```

Comment:
Would it make sense to use `-U` instead of `-g`?

Reply:
I don't think so. the behavior of -U would mean the path would persist even if the user removed the config file, and it has no real benefit since the config is run at shell creation anyways.

Reply:
Great, thanks for the info.